Intersect.Client/Entities/Player.cs

```diff
@@ -1534,57 +1549,104 @@ public bool TryBlock()
 
         public bool TryAttack()
         {
-            if (AttackTimer > Timing.Global.Milliseconds || IsBlocking || (IsMoving && !Options.Instance.PlayerOpts.AllowCombatMovement))
+            if (AttackTimer > Timing.Global.Milliseconds || IsBlocking ||
+                (IsMoving && !Options.Instance.PlayerOpts.AllowCombatMovement))
             {
                 return false;
             }
 
             int x = Globals.Me.X;
             int y = Globals.Me.Y;
             var map = Globals.Me.MapId;
+
+            CheckAdjacentTiles();
```

Comment:
this is what the previous PR did with the hitboxes login but extracted as a method instead for better readability. do you mean to move those over to the switch (Globals.Me.Dir) like it was on the previous PR or you mean to completly purge this logic ?

Reply:
this is what the previous PR did with the hitboxes logic but extracted as a method instead for better readability. do you mean to move those over to the switch (Globals.Me.Dir) like it was on the previous PR or you mean to completly purge this logic that helps when targeting diagonally on nearby entities and such? if we remove this, people opting out from using auto-face to target (auto change dir to target) won't be able to do this with ease, so far this is a comment i got from some devs ; i don't like this logic too much either since auto-face to target does cover this scenario and this is pretty much the same but less efficient in many ways ... but apparently some people wants the beneficts of being able to do this only with nearby targets only (?)

idk, if people want to exclude auto face to target from their projects, they shouldn't advocate for this logic either, i would honestly remove this logic since auto face to target already covers it, perhaps, a max distance config for auto face to target could be added instead and then this wouldn't be required at all anymore ❓ but then, i'm pretty sure that targeting itself had a range limit that can be used.

Reply:
wiped this logic out, instead, i've added a way to limit down autoface to target's ability so it doesn't turns around the character to it's opposite directions (i.e : turn around from up to down by targetting someone behind us: some people have commented that this is an unwanted behavior in some games - while there's other vast majority likes it, thus i'm making it configurable).